cfme/containers/provider/__init__.py

```diff
@@ -355,6 +361,45 @@ def step(self):
         sel.click(InfoBlock('Overview', 'Topology'))
 
 
+class AdHocMetricsView(BaseLoggedInPage):
+    filter_dropdown = SelectorDropdown('uib-tooltip', 'Filter by')
+    apply_btn = form_buttons.apply_filters
```

Comment:
@mfalesni , @quarckster 
when i am trying to use `widgetastic_patternfly.Button` but i am missing something.

```from widgetastic_patternfly import Button
apply_btn = Button("Apply Filters")
apply_btn.click()
```

This it the result of this code:
```
>>> apply_btn.klass.fill()
Traceback (most recent call last):
  File "/home/snaim/virtEns/myDevEnv/lib/python2.7/site-packages/IPython/core/interactiveshell.py", line 2882, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-9-c01fc0ac30e8>", line 1, in <module>
    apply_btn.klass.fill()
TypeError: unbound method fill() must be called with Button instance as first argument (got nothing instead)
```

Can you tell me what i am doing wrong?